test/extended/util.go

```diff
@@ -0,0 +1,23 @@
+package extended
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+)
+
+var testContext = e2e.TestContextType{}
+
+func kubeConfigPath() string {
+	return os.Getenv("KUBECONFIG")
+}
+
+func writeTempJSON(path, content string) error {
```

Comment:
Accepting a `runtime.Object` will make this utility easier to use.  As it is, there's no reason to expect JSON to be delivered to this method.